### DISCUSSION TITLE
What is the world's largest ocean?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#818] [Galaxy Trivia] [answered]
**Bonus:** What percentage of the earth's total water does each ocean hold?

**Comment by ParanoidUser** [comment]
@zer0Kerbal / @TheVigor, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment]
The world's largest ocean is the **Pacific Ocean**. It covers an area of approximately 63,800,000 square miles (165,250,000 square kilometers), making it significantly larger than any other ocean on Earth. In terms of the percentage of the world's water it holds, the Pacific Ocean contains about **50% of the Earth's total water**.

**Comment by SANKARAMDAS** [comment]
The Pacific Ocean is the largest and deepest of the world ocean basins. Covering approximately 63 million square miles and containing more than (50%) half of the free water on Earth, the Pacific is by far the largest of the world's ocean basins. All of the world's continents could fit into the Pacific basin.

https://oceanservice.noaa.gov/facts/biggestocean.html#:~:text=The%20Pacific%20Ocean%20is%20the,of%20the%20world%20ocean%20basins.&text=Covering%20approximately%2063%20million%20square,fit%20into%20the%20Pacific%20basin.

**Comment by ParanoidUser** [comment]
The Pacific Ocean is indeed the largest. If it holds about 50% of the free water on Earth, what are the percentages for the other oceans?

**Reply by peterbaccaro** [reply]
The approximate percentages of water in each ocean are as follows:

- **Pacific Ocean**: The Pacific Ocean holds about **50.1%** of the Earth's total water volume.
- **Atlantic Ocean**: The Atlantic Ocean contains approximately **23.5%** of the Earth's total water volume.
- **Indian Ocean**: The Indian Ocean comprises around **19.5%** of the Earth's total water volume.
- **Southern Ocean**: The Southern Ocean encompasses about **4.8%** of the Earth's total water volume.
- **Arctic Ocean**: The Arctic Ocean holds approximately **3.7%** of the Earth's total water volume.

**Reply by SANKARAMDAS** [reply, marked as answer]
| Ocean                    | Area (km)2         | Water  Volume (km)3 | Avg depth (m) |
|--------------------------|--------------------|---------------------|---------------|
| Pacific Ocean            | 168,723,000(46.6%) | 669,880,000(50.1%)  | 3,970         |
| Atlantic Ocean           | 85,133,000(23.5%)  | 310,410,900(23.3%)  | 3,646         |
| Indian Ocean             | 70,560,000(19.5%)  | 264,000,000(19.8%)  | 3,741         |
| Antarctic/Southern Ocean | 21,960,000(6.1%)   | 71,800,000(5.4%)    | 3,270         |
| Arctic Ocean             | 15,558,000(4.3%)   | 18,750,000(1.4%)    | 1,205         |

https://en.wikipedia.org/wiki/Ocean

**Comment by zer0Kerbal** [comment]
Depends if you are counting ice as land.... as good of question is what is the largest sea?

Actually - I do believe a larger ocean has been found - but it is subterranean.
